DFReader.py

```diff
@@ -555,6 +555,103 @@ def set_message_timestamp(self, m):
         m._timestamp = self.timebase + count/rate
 
 
+class DFMetaData(object):
+    '''handle dataflash messages metadata'''
+    def __init__(self,parent):
```

Comment:
```suggestion
    def __init__(self, parent):
```

there are similar formatting things throughout this Python code.  At some stage it would be nice to have this stuff `flake8`-clean, but if we could generally avoid introducing more bad stuff that would be nice.  No need to go back and do so on this PR.